templates/login.html

```diff
@@ -6,12 +6,18 @@
     <h1>Hello, world!</h1>
     <p>You need to created an account to be able to submit your solutions to the
     coding problems. Since you are a coder, I'm pretty sure you have a Github
-    account, right?</p>
+    account, right? If,No thats Ok you can login through google.</p>
```

Comment:
@nishutosh Although I have made the changes, please look after those grammatical errors. :)